Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:trusty
+
+LABEL description="Insight API (Decred fork)"
+LABEL version="1.0"
+LABEL maintainer "peter@prioritylane.com"
+
+ENV NODE_VERSION v0.10.40
+ENV TERM linux
+
+# update base distro & install build tooling
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -qy build-essential python git curl
+
+ENV HOME /root
+ENV APPDIR $HOME/insight
+ENV NVM_DIR $HOME/.nvm
+
+RUN mkdir -p $APPDIR
+
+WORKDIR $APPDIR
+
+COPY . $APPDIR
+
+# install & configure NodeJS
+RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash && \
```

Comment:
Any way we can avoid this curl to a remote script?  We typically prefer to avoid these types of calls.